version/version.go

```diff
@@ -21,7 +21,7 @@ import (
 // The presence and format of this constant is very important.
 // The debian/rules build recipe uses this value for the version
 // number of the release package.
-const version = "2.9-beta1"
+const version = "2.9-beta3"
```

Comment:
If you rebase on top of current develop/HEAD you should get this.

Reply:
This is what I see in the develop branch `const version = "3.0-beta1"`. I'm not sure what you mean by this?

Reply:
Ah yes, sorry. That was bumped when we created the 2.9 branch. You can keep the 3.0-beta1. When bootstrapping, juju will not find any agent binaries using the streams and fall back to the ones you have compiled locally.

Reply:
p.s. remember that I had a problem with the controller not being able to find the implementation of packet provider so we had to trick it to pull the binary that had that implementation.

Reply:
I am not sure why this was causing issues with you and will need to test locally. The idea is that when you bootstrap, if the client cannot find the appropriate agent binary (e.g. if you work on a local juju tree), it will show a warning and upload the binary you have just compiled.

So it's pretty much: `make install; juju bootstrap...`. Note that this assumes that your local dev env is running the same OS _type_/arch combo (so if you are compiling on mac/windows and bootstrapping on a linux instance this will not work out of the box)

Reply:
So I tried as described and I ended up with this error:
```
ERROR cannot read agent metadata in directory /var/lib/juju/tools/3.0-beta1.1-bionic-amd64: open /var/lib/juju/tools/3.0-beta1.1-bionic-amd64/downloaded-tools.txt: no such file or directory
2020-12-12 03:11:46 DEBUG cmd supercommand.go:537 error stack: 
cannot read agent metadata in directory /var/lib/juju/tools/3.0-beta1.1-bionic-amd64: open /var/lib/juju/tools/3.0-beta1.1-bionic-amd64/downloaded-tools.txt: no such file or directory
```